.babelrc

```diff
@@ -8,6 +8,7 @@
     "transform-es2015-spread",
     "transform-flow-strip-types",
     "transform-async-to-generator",
+    "syntax-trailing-function-commas",
```

Comment:
this is helpful to support some issues in node and does not generally affect the browsers